### ISSUE TITLE
No territories on mobile

### ISSUE THREAD

**Issue by pycanis** [stackernews/stacker.news#1260]
**Description**
I don't see any territories when I want to create a post.

**Screenshots**
![Screenshot_20240708-175818](https://github.com/stackernews/stacker.news/assets/141557160/1c2c5bfe-f222-4ef2-8c4c-8e4b973a4de4)

**Environment:**
If you only experience the issue on certain devices or browsers, provide that info.
 - Device: Pixel 7
 - OS: GrapheneOS
 - Browser: Vanadium
 - Version: 126.0.6478.122.2

**Comment by ekzyis** [comment]
Is that a new, consistent issue or is it the first time you use SN on mobile?

We fetch the territory list, so maybe that query failed on your device for some reason. But it shouldn't at least fail always.

**Comment by pycanis** [comment]
> Is that a new, consistent issue or is it the first time you use SN on mobile?
> 
> We fetch the territory list, so maybe that query failed on your device for some reason. But it shouldn't at least fail always.

I've been using it on mobile for some time but I don't remember it always being like this. It's been happening for a few days at least. I'll connect my phone to PC and have a look at the console.

**Comment by ekzyis** [comment]
> It's been happening for a few days at least.

So you mean it's not fixed by reloading the page?

> I'll connect my phone to PC and have a look at the console.

That would be cool and much appreciated!

Since it might not be easy to find the sub query among all the other GraphQL queries, here's a tip:

When you have the network tab open, press CTRL+F. That will open a separate search window where you can now search through response bodies. Try `"__typename":"Sub"`.

That should give you the network request to fetch the territories:

![2024-07-08-193407_1920x1080_scrot](https://github.com/stackernews/stacker.news/assets/27162016/89647a67-a189-47fa-8712-3b98dabd56b6)

_Mhh, but if your query indeed failed, then I guess you won't find it like this :thinking:_

**Comment by pycanis** [comment]
Okay, so I've got it figured out. In the console, there was `ReferenceError: WebAssembly is not defined` error. Is SN using WA in any way?

I've ran into the same issue with my app that actually uses WA and I had to enable JavaScript JIT permission. With that on, everything's working fine. It was actually blocking much more stuff than just the territories query.

Just found out [the reason why it started happening](https://grapheneos.social/@GrapheneOS/112708050929472969).

![Screenshot_20240708-194953](https://github.com/stackernews/stacker.news/assets/141557160/566afb63-b582-4b25-8c30-5a4ad2307ecc)
